### PR TITLE
Remove use of soon to be removed css option

### PR DESCRIPTION
**Pull request by benmccann**
This option is being removed in https://github.com/sveltejs/rollup-plugin-svelte/pull/147

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "@rollup/plugin-commonjs": "^14.0.0",
     "@rollup/plugin-node-resolve": "^8.0.0",
     "rollup": "^2.3.4",
+    "rollup-plugin-css-only": "^3.0.0",
     "rollup-plugin-livereload": "^2.0.0",
     "rollup-plugin-svelte": "^6.0.0",
```
Comment (lukeed) [inline package.json:15]:
```suggestion
    "rollup-plugin-svelte": "^7.0.0",
```

**Comment by lukeed** [comment]
Considering this as a "draft" PR until `7.0` is out later today/tomorrow

File: package.json (hunk 1 above)
Comment (benmccann) [reply package.json]: done. I also removed `emitCss` from `rollup.config.js` so that we can rely on the default

File: rollup.config.js [hunk 2]
```diff
@@ -38,13 +39,11 @@ export default {
 	plugins: [
 		svelte({
 			// enable run-time checks when not in production
-			dev: !production,
-			// we'll extract any component CSS out into
-			// a separate file - better for performance
-			css: css => {
-				css.write('bundle.css');
-			}
+			dev: !production
```
Comment (lukeed) [inline rollup.config.js:42]: If we're updating this for 7.0, this won't work anymore. Should be
```suggestion
			compilerOptions: {
			  dev: !production
			}
```

**Review by lukeed — APPROVED** [review]

**Pull request merged**